spec/chromium-spec.js

```diff
@@ -1023,7 +1025,23 @@ describe('chromium feature', function () {
       w.webContents.loadURL(pdfSource)
     })
 
+    it('should download a pdf when plugins are disabled', function (done) {
+      createBrowserWindow(false)
+      ipcRenderer.sendSync('set-download-option', false, false)
+      ipcRenderer.once('download-done', function (event, state, url,
```

Comment:
I don't think you need to wrap the arguments here, long lines are fine in specs.

If you do want to wrap, can you align the `mimeType` param with the `event` param?